conference/react-js/src/components/Talks/ReviewTalk/ReviewTalk.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ReviewTalk = ({ close }) => (
+  <div className="pop-up-wrapper">
+    <div className="pop-up review-talk">
+      <div className="review-talk__topic">
+        JVM Languages and new programming paradigms
+      </div>
+      <div className="review-talk__type">
+        Regular Talk:
+      </div>
+      <div className="review-talk__title">
+        Why Java Sucks
+      </div>
+      <p className="review-talk__description">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore
```

Comment:
is it necessary to use mock data in implementation? I think it should be removed.

Reply:
I need this data just to review pop-up design